joomla/joomla-cms
# [4.0] Fix actionlog message value

Pull Request for Issue #24846 .

### Summary of Changes
Load language on after initialise.


### Testing Instructions
- Enable/Disable a plugin.
- Add/Update/Delete an extension, an article, an user, a group.
- Publish/Unpublish/Trash/Archive an article, a categorie, a menu.
- See the Latest Actions module in the Control Panel

### Expected result
No constant value, the actionlog message is translated.


### Actual result
As expected result.


### Documentation Changes Required
